### PR TITLE
Docs: update behaviour with mimir/loki managed rules and namespaces

### PR DESCRIPTION
**Pull request by bastischubert**
**What is this feature?**

Updated docs to reflect the current behaviour of grafana
**Why do we need this feature?**

Correct Documentation
**Who is this feature for?**

everyone

**Which issue(s) does this PR fix?**:

none

Fixes #

**Special notes for your reviewer:**

Please check that:
- [x] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [x] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: docs/sources/alerting/alerting-rules/create-mimir-loki-managed-recording-rule.md [hunk 1]
```diff
@@ -43,8 +43,8 @@ To create a Grafana Mimir or Loki managed recording rule
    - Select your Loki or Prometheus data source.
    - Enter a PromQL or LogQL query.
 1. In Step 3, add the namespace and the group.
-   - From the **Namespace** dropdown, select an existing rule namespace. Otherwise, click Add new and enter a name to create a new one. Namespaces can contain one or more rule groups and only have an organizational purpose. For more information, see [Grafana Mimir or Loki rule groups and namespaces]({{< relref "edit-mimir-loki-namespace-group/" >}}).
-   - From the **Group** dropdown, select an existing group within the selected namespace. Otherwise, click **Add new** and enter a name to create a new one.
+   - From the **Namespace** dropdown, select an existing rule namespace. Otherwise, type the desired name and hit Enter to create a new one. Namespaces can contain one or more rule groups and only have an organizational purpose. For more information, see [Grafana Mimir or Loki rule groups and namespaces]({{< relref "edit-mimir-loki-namespace-group/" >}}).
+   - From the **Group** dropdown, select an existing group within the selected namespace. Otherwise, type the desired name and hit Enter to create a new one.
```
Comment (brendamuir) [inline docs/sources/alerting/alerting-rules/create-mimir-loki-managed-recording-rule.md:47]:
```suggestion
   - From the **Namespace** dropdown, select an existing rule namespace or enter a new one.  Namespaces can contain one or more rule groups and only have an organizational purpose. For more information, see [Grafana Mimir or Loki rule groups and namespaces]({{< relref "edit-mimir-loki-namespace-group/" >}}).
   - From the **Group** dropdown, select an existing group within the selected namespace or create a new one.
```

File: docs/sources/alerting/alerting-rules/create-mimir-loki-managed-rule.md [hunk 2]
```diff
@@ -48,8 +48,8 @@ _Refer to [Add a Grafana Mimir or Loki managed alerting rule]({{< relref "#add-a
...
+   - From the **Namespace** dropdown, select an existing rule namespace. Otherwise, type the desired name and hit Enter to create a new one. Namespaces can contain one or more rule groups and only have an organizational purpose. For more information, see [Grafana Mimir or Loki rule groups and namespaces]({{< relref "edit-mimir-loki-namespace-group/" >}}).
+   - From the **Group** dropdown, select an existing group within the selected namespace. Otherwise, type the desired name and hit Enter to create a new one. Newly created rules are appended to the end of the group. Rules within a group are run sequentially at a regular interval, with the same evaluation time.
    - Add a description and summary to customize alert messages. Use the guidelines in [Annotations and labels for alerting]({{< relref "../fundamentals/annotation-label/" >}}).
    - Add Runbook URL, panel, dashboard, and alert IDs.
 1. In Step 5, add custom labels.
```
Comment (brendamuir) [inline docs/sources/alerting/alerting-rules/create-mimir-loki-managed-rule.md:55]:
```suggestion
   - From the **Namespace** dropdown, select an existing rule namespace or enter a new one. Namespaces can contain one or more rule groups and only have an organizational purpose. For more information, see [Grafana Mimir or Loki rule groups and namespaces]({{< relref "edit-mimir-loki-namespace-group/" >}}).
   - From the **Group** dropdown, select an existing group within the selected namespace or enter a new one. Newly created rules are appended to the end of the group. Rules within a group are run sequentially at a regular interval, with the same evaluation time.
   - Add a description and summary to customize alert messages. Use the guidelines in [Annotations and labels for alerting]({{< relref "../fundamentals/annotation-label/" >}}).
   - Add Runbook URL, panel, dashboard, and alert IDs.
1. In Step 5, add custom labels.
```

**Review by brendamuir — APPROVED** [review]

**Review by brendamuir — APPROVED** [review]

File: docs/sources/alerting/alerting-rules/create-mimir-loki-managed-rule.md [hunk 3]
```diff
@@ -50,9 +50,9 @@ If you do not want to manage alerting rules for a particular Loki or Prometheus
...
-   - From the **Namespace** dropdown, select an existing rule namespace. Otherwise, click **Add new** and enter a name to create a new one. Namespaces can contain one or more rule groups and only have an organizational purpose. For more information, see [Grafana Mimir or Loki rule groups and namespaces]({{< relref "./edit-mimir-loki-namespace-group" >}}).
-   - From the **Group** dropdown, select an existing group within the selected namespace. Otherwise, click **Add new** and enter a name to create a new one. Newly created rules are appended to the end of the group. Rules within a group are run sequentially at a regular interval, with the same evaluation time.
-   - Add a description and summary to customize alert messages. Use the guidelines in [Annotations and labels for alerting]({{< relref "../fundamentals/annotation-label" >}}).
+   - From the **Namespace** dropdown, select an existing rule namespace or enter a new one. Namespaces can contain one or more rule groups and only have an organizational purpose. For more information, see [Grafana Mimir or Loki rule groups and namespaces]({{< relref "edit-mimir-loki-namespace-group/" >}}).
```
Comment (jdbaldry) [inline docs/sources/alerting/alerting-rules/create-mimir-loki-managed-rule.md:53]:
```suggestion
   - From the **Namespace** dropdown, select an existing rule namespace or enter a new one. Namespaces can contain one or more rule groups and only have an organizational purpose. For more information, see [Grafana Mimir or Loki rule groups and namespaces]({{< relref "./edit-mimir-loki-namespace-group" >}}).
```

**Review by jdbaldry — COMMENTED** [review]
You will also need to run `prettier` on these files.

If you would like, I can run `prettier` and commit the changes before merge once the `doc-validator` suggestions have been applied.

**Comment by brendamuir** [comment]
Hi there-thank you very much for your contribution! I've incorporated your changes into a PR: https://github.com/grafana/grafana/pull/69997 and it's been merged. Closing this PR now.
Thanks again!

**Pull request closed without merging**